util/is-inverse-benchmark.js

```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ * @param {{}} benchmark
+ * @returns {boolean}
+ */
+var isInverseBenchmark = function isInverseBenchmark(benchmark) {
```

Comment:
Oh nice you already have this! Yeah - I'd suggest using this function in `convert-qpp-to-measures` instead of persisting the map.